### ISSUE TITLE
mIoU Calculation Cityscapes, class weights

### ISSUE THREAD

**Issue by biboffff** [tensorflow/models#5863]
### System information
- **What is the top-level directory of the model you are using**: Deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.12
- **CUDA/cuDNN version**: 9.0

Do you use any class weights for the mIoU calculation for the cityscapes validation set?

I try to calculate the IoU per image with a custom python script. For a tfrecord containing a single image I got the exact confusion matrix as tensorflow calculates during validation, however the final/printed mIoU value differs from mine.

best regards,
chris

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Bazel version
GPU model and memory
Exact command to reproduce

**Comment by biboffff** [comment]
Thank you. They are not relevant/not available.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
